src/main/kotlin/app/revanced/patches/music/audio/codecs/fingerprints/CodecsLockFingerprint.kt

```diff
@@ -22,7 +22,7 @@ object CodecsLockFingerprint : MethodFingerprint(
         Opcode.INVOKE_STATIC,
         Opcode.MOVE_RESULT_OBJECT,
         Opcode.INVOKE_INTERFACE,
-        Opcode.INVOKE_DIRECT,
+        Opcode.INVOKE_VIRTUAL,
```

Comment:
Can you assure, this fingerprint will work with previous versions mentioned in the compatible versions list as well?

Reply:
I checked the two latest and the first version, so I'm guessing that they all will work.